src/sentry/api/endpoints/project_transaction_threshold.py

```diff
@@ -90,7 +90,7 @@ def post(self, request: Request, project) -> Response:
                 )
                 project_threshold.threshold = data.get("threshold") or project_threshold.threshold
                 project_threshold.metric = data.get("metric") or project_threshold.metric
-                project_threshold.edited_by = request.user
+                project_threshold._id = request.user.id
```

Comment:
```suggestion
                project_threshold.edited_by_id = request.user.id
```

I don't think the `_id` attribute is what you want here.